parser-typechecker/src/Unison/Codebase/Editor/Output.hs

```diff
@@ -287,7 +287,7 @@ isFailure o = case o of
   Typechecked{} -> False
   DisplayDefinitions _ _ m1 m2 -> null m1 && null m2
   DisplayRendered{} -> False
-  TodoOutput _ todo -> TO.todoScore todo /= 0 && not (TO.noConflicts todo)
+  TodoOutput _ todo -> TO.todoScore todo > 0 || not (TO.noConflicts todo)
```

Comment:
this was just wrong! should be `||` not `&&`. It's a failure if EITHER there's conflicts or there is more todo.